### PR TITLE
PR 1 of 4 for SNV consensus: Set Up

### PR DESCRIPTION
**Pull request by cansavvy**
### Purpose/implementation

This is a side analysis that originated from [this original PR](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/174) but after some reorganization has been split up and spread out. 

This first PR sets up items for the rest and includes: 
- CircleCI test changes 
   - VAF experiment is commented out
- Dockerfile changes 
   - UpSetR is installed through CRAN
- README changes
   - Added documentation for the upcoming scripts - there will be more to come for sure. 
- "I-re-ran-everything-through-a-linter" changes

### Issue

For SNV caller comparison #161, #103 and Tumor Mutation Burden #3 and sort of #11

### Directions for reviewers

I wouldn't spend too much time on the README stuff, because depending on the upcoming reviews things may change, however, I want reviewers to get a bit of an overview of what's to come. 

Do these changes make sense overall?

### Results

This script doesn't add results, but is a step toward getting the snv consensus. 

#### Docker and continuous integration

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
Main changes to Dockerfile:
- UpSetR - CRAN version now. 
- [x] This analysis has been added to continuous integration.

#### PR Checklist
* [x] Run a linter
* [x] Set the seed- I didn't, but let me know if I should.
* [x] Comments and/or documentation up to date 
* [x] Double check your paths
* [x] Spell check any Rmd file or md file
* [x] Restart R and run all notebooks fresh and save

### CODE REVIEW

File: analyses/snv-callers/README.md [hunk 1]
```diff
@@ -135,10 +192,10 @@ vaf = (t_alt_count) / (t_ref_count + t_alt_count)
 ```
 This is following the [code used in
 `maftools`](https://github.com/PoisonAlien/maftools/blob/1d0270e35c2e0f49309eba08b62343ac0db10560/R/plot_vaf.R#L39).
-The VAF calculations and other special variables are added to the MAF fields and written to a TSV ending in `_vaf.tsv` in the caller's results folder.
+The VAF calculations and other special variables are added to the MAF fields and written to a TSV ending in `_vaf` in the caller's results folder.
```
Comment (jashapiro) [inline analyses/snv-callers/README.md]:
```suggestion
The VAF calculations and other special variables are added to the MAF fields and written to a file ending in `_vaf` in the caller's results folder.
```

File: analyses/snv-callers/README.md [hunk 2]
```diff
@@ -206,38 +267,46 @@ In order to compare mutations across callers, I created a `mutation_id` from com
...
+│       │   │   └── consensus_mutation.maf.tsv
 │       │   ├── lancet
-│       │   │   ├── lancet_vaf.tsv
-│       │   │   ├── lancet_vaf.tsv.zip
-│       │   │   ├── lancet_tmb.tsv
-│       │   │   ├── lancet_tmb.tsv.zip
-│       │   │   ├── lancet_region.tsv
-│       │   │   ├── lancet_region.tsv.zip
+│       │   │   ├── lancet_vaf.rds
+│       │   │   ├── lancet_vaf.rds.zip
```
Comment (jashapiro) [inline analyses/snv-callers/README.md]: Do you have both the file and a zipped version? (I would assume the .rds is compressed or not internally)

File: analyses/snv-callers/README.md [hunk 3]
```diff
@@ -206,38 +267,46 @@ In order to compare mutations across callers, I created a `mutation_id` from com
...
 OpenPBTA-analysis
 ├── analyses
 │   └── snv-callers
-│       ├── run_variant_caller_reports.R
+│       ├── run_caller_analysis.sh
+│       ├── compare_snv_callers.Rmd
 │       ├── scripts
 │       │   ├── 00-set-up.R
 │       │   ├── 01-calculate_vaf_tmb.R
 │       │   └── 02-run_eval.R
```
Comment (jashapiro) [inline analyses/snv-callers/README.md]: add 03 and 04, since they are already in this doc.

File: analyses/snv-callers/README.md (hunk 2 above)
Comment (cansavvy) [reply analyses/snv-callers/README.md]: You're right. This is redundant/inaccurate.

**Pull request merged**